### PR TITLE
./test-backend.sh detects and runs all tests

### PR DESCRIPTION
**Pull request by pedjak**
No need to manage per hand a list of packages having tests

### CODE REVIEW

File: test-backend.sh [hunk 1]
```diff
@@ -20,8 +20,8 @@ export GOFLAGS="-mod=vendor"
 # Invoke ./cover for HTML output
 COVER=${COVER:-"-cover"}
 
-TESTABLE="pkg/auth pkg/proxy pkg/server pkg/helm/actions pkg/helm/handlers"
-FORMATTABLE="${TESTABLE} cmd/bridge pkg/version"
+TESTABLE="./..."
```
Comment (benjaminapetersen) [inline test-backend.sh]: Perfect, given I just left [this comment](https://github.com/openshift/console/pull/4362#pullrequestreview-361425647)!  👍

**Comment by pedjak** [comment]
@benjaminapetersen `e2e-gcp-console` is failing constantly on this PR although the change is fully unrelated, blocking the merge. Can we do something about it?

**Pull request merged**